modules/mvt/src/lib/mapbox-vector-tile/vector-tile.ts

```diff
@@ -10,6 +10,13 @@ export default class VectorTile {
   }
 }
 
+/**
+ *
+ * @param tag
+ * @param layers
+ * @param pbf
+ * @returns {void}
+ */
 function readTile(tag: number, layers?: {[x: string]: VectorTileLayer}, pbf?: Protobuf) {
```

Comment:
Let's move `void` into the types and remove from JSDoc (in all places!)
```suggestion
 */
function readTile(tag: number, layers?: {[x: string]: VectorTileLayer}, pbf?: Protobuf): void {
```